doc/source/examining/loading_data.rst

```diff
@@ -2111,6 +2111,33 @@ For example, to add support for a longint field named
 ``my_custom_field``, one would add ``('my_custom_field', 'l')`` to ``assoc``.
 
 
+Particle ages and formation times
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For cosmological simulations that include star particles, RAMSES stores particle
+formation times as conformal times. To access these raw data in conformal units
+use the ``"conformal_formation_time"`` field. This will return the formation
```

Comment:
And what should the user use in the case of a non-cosmological simulation? From what I understood, their is not `conformal_formation_time` in this case.